src/flows/data_retention/development_snowflake_schemas.py

```diff
@@ -0,0 +1,42 @@
+"""DELETE all DEVELOPMENT DB schemas older than 89 days
+
+Reference: https://getpocket.atlassian.net/wiki/spaces/CP/pages/2703949851/Snowflake+Data+Deletion+and+Retention
+"""
+from prefect import Flow, task, unmapped
+from prefect.schedules import Schedule
+from prefect.schedules.clocks import CronClock
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery
+from utils.flow import get_flow_name
+
+FLOW_NAME = get_flow_name(__file__)
+DATABASE = "DEVELOPMENT"
```

Comment:
@kennylong This raises a good question for some one-off static tables that we have located in this schema, specifically the scrappy AB test for Android push notifications targeting TWAU-adjacent users. Do we want to move that table elsewhere (e.g. codify it as a dbt model)? Or are we fine losing that table after 90 days?

Reply:
@kennylong Gotcha, thank you for the context! Do we know where these datasets need to end up? i.e. is there a different database I should immediately advise Sreejita to move the AB test assignment table to?

Reply:
Before asking Sreejita to move the table, I'd like to get @jeshuaborges's take on the above proposal, which boils down to:

* Create a new to-be-named database meant for holding longer lasting, manually created tables
* Perhaps have some PR process to formalize/document how we create/move tables into there
* Include a PR step in the template to add to add an accompanying `DELETE` to the delete .sql file

Reply:
> * Create a new to-be-named database meant for holding longer lasting, manually created tables
> * Perhaps have some PR process to formalize/document how we create/move tables into there
> * Include a PR step in the template to add to add an accompanying `DELETE` to the delete .sql file

Additional thoughts:
- The schema created in this new DB (or perhaps we could have this setup even for the `DEVELOPMENT` DB?) has an expiration date set (unlike BigQuery, Snowflake does not have such a build-in feature)
- Maintain a table with an entry for each schema for an expiration date
- Schema will be deleted  at the expiration date (schema without an expiration date set will be deleted the next day or some default x days after the creation date)